kong/api/routes/clustering.lua

```diff
@@ -0,0 +1,18 @@
+local clustering = require("kong.clustering")
+local kong = kong
+
+
+return {
+  ["/clustering/status"] = {
+    GET = function(self, db, helpers)
+      if kong.configuration.role ~= "admin" then
+        return kong.response.exit(400, {
+          message = "this endpoint is only available when Kong is " ..
+                    "configured to run as Control Plane for the cluster"
```

Comment:
We should be consistent. If the role is `admin` then we should have this message should have that.

My personal opinion is that we should have role as `control-plane` or `controller` or some such as the `role`.